src/views/ProductInventoryDetail.vue

```diff
@@ -0,0 +1,779 @@
+  globeOutline,
+  listOutline,
+  locationOutline,
+  shareOutline,
+  shirtOutline,
+  sendOutline,
+  storefrontOutline,
+  syncOutline,
+  ticketOutline,
+} from 'ionicons/icons';
+import Image from '@/components/Image.vue';
+import EditQuantityModal from '@/components/EditQuantityModal.vue';
+import LocationPopover from '@/components/LocationPopover.vue';
+import PurchaseOrderPopover from '@/components/PurchaseOrderPopover.vue';
+import FulfillmentSettingsPopover from '@/components/FulfillmentSettingsPopover.vue';
+
+export default defineComponent({
+  name: 'ProductInventoryDetail',
+  components: {
+    IonBackButton,
+    IonButton,
+    IonButtons,
+    IonCard,
+    IonCardHeader,
+    IonCardTitle,
+    IonCheckbox,
+    IonChip,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonNote,
+    IonPage,
+    IonSearchbar,
+    IonSegment,
+    IonSegmentButton,
+    IonTitle,
+    IonThumbnail,
+    IonToggle,
+    IonToolbar,
+    Image
+  },
+  methods: {
+    segmentChanged(ev: CustomEvent) {
+      this.segment = ev.detail.value;
+    },
+    async editQuantity() {
+      const editmodal = await modalController.create({
+        component: EditQuantityModal
+      });
+      return editmodal.present();
+    },
+    async openLocationPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: LocationPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+    async openPopover(ev: Event) {
+      const popover = await popoverController.create({
+        component: PurchaseOrderPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+    async openFulfillmentSettings(ev: Event) {
+      const popover = await popoverController.create({
+        component: FulfillmentSettingsPopover,
+        event: ev,
+        translucent: true,
+        showBackdrop: false,
+      });
+      return popover.present();
+    },
+  },
+  setup() {
+    const router = useRouter();
+    const segment = ref("locations");
+
+    return {
+      businessOutline,
+      calendarOutline,
+      checkmarkOutline,
+      ellipsisVerticalOutline,
+      globeOutline,
+      listOutline,
+      locationOutline,
+      shareOutline,
+      shirtOutline,
+      sendOutline,
+      storefrontOutline,
+      syncOutline,
+      ticketOutline,
+      router,
+      segment,
+    }
+  }
+});
+</script>
+
+<style scoped>
+.product {
+  display: grid;
+  grid-template-columns: 254px auto;
+  justify-items: start;
+  align-items: center;
+}
+
+.product-info,
+.variant-info {
+  width: 343px;
+}
+
+.variant,
+.actions {
+  display: grid;
+  grid-template-columns: repeat(2, auto);
+  justify-content: space-between;
+  align-items: center;
+}
+
+.open-orders > ion-card > ion-item > ion-label:not(:first-child)  {
```

Comment:
I think we can make this just for ion-item and select not first child and not last child.

Reply:
Do we want our last child also to be aligned at start ?
Do we only want middle child to aligned center ?

Reply:
now that selector is updated, please remove classes from above